internal/host/validator.go

```diff
@@ -502,10 +502,10 @@ func (v *validator) printBelongsToMachineCidr(c *validationContext, status Valid
 		if swag.StringValue(c.cluster.Kind) == models.ClusterKindAddHostsCluster {
 			return "No machine network CIDR validation needed: Day2 cluster"
 		}
-		return fmt.Sprintf("Host belongs to machine network CIDR %s", network.GetMachineCidrById(c.cluster, 0))
+		return "Host belongs to all machine network CIDRs"
 	case ValidationFailure:
 		if network.IsMachineCidrAvailable(c.cluster) {
```

Comment:
No need to have 2 different messages now. You can remove the  `IsMachineCidrAvailable` here.

Reply:
Done